CONTRIBUTING.md

```diff
@@ -0,0 +1,324 @@
+
+### eval
+
+`eval.py` should contain a working implementing of the model evaluation process. This script should load the trained model (using the information in `config/config.json`) and evaluate it against the evaluation datasets. The evaluation metrics reported should correspond to that reported in the `README.md`.
+
+### utils
+
+`utils.py` should contain other functions which are useful for `train.py` or `eval.py` but do not directly fit within any of the other scripts above.
+
+### README
+
+The `README` for the model should provide a concise introduction to the model. The following information are required:
+
+- Citation and link to the original paper or source that introduced the model
+- Citation and link to the train, validation and test datasets that the model was trained on. If the model was trained and evaluated on licensed datasets, information should be provided on how the SG-NLP team can obtain access to the evaluation (test) dataset. The train dataset may be omitted.
+- Evaluation metrics. Please cite the appropriate paper if the evaluation metric is a published benchmark.
+- Model size (in terms of the size of the trained model's weights)
+- Training information such as hyperparameter values and compute time and resources used to train the model.
+
+### Model weights and artefacts
+
+Model weights and artefects comprise of:
+
+- Saved model weights. Specifically, the `pytorch_model.bin` file saved using the `save_pretrained` method from the model class implemented in `modeling.py`. For now, only models implemented in `PyTorch` are accepted. The team is looking into accepting models implemented in `TensorFlow` as well.
+- Model config. The `config.json` generated when using the `save_pretrained` method from the model config class implemented in `config.py`.
+- Any artefacts needed by the tokenizer or preprocessor.
+
+## Submitting Bug Fixes
+
+If you spotted a bug, please follow these steps to report them.
+
+1. Check the issues list to see whether the bug has already been reported. If an issue has already been created, please comment on that issue with details on how to replicate the bug.
+2. If there is no issue relevant to the bug, please open a GitHub with
+   - Clear description of the bug
+   - Information related to your environment (ie, OS, package version, Python version, etc)
+   - Steps on how to replicate the bug (ie, a code snippet that we could run to encounter the bug)
+
+## Adding Documentation
+
+One easy way to contribute is to add or refine documentation / docstrings to the models that are currently available. `sgnlp` uses [Sphinx](https://www.sphinx-doc.org/en/master/) to generate our documentation pages. Please refer to the Sphinx documentation on the formatting expected in the docstrings. Once the docstrings have been added or edited, please submit a pull request.
```

Comment:
Actually the docstring format we follow is the google style python docstring. Sphinx supports this format, but I don't think Sphinx itself indicates anywhere how this format is like.